### PR TITLE
Initial connect

### PR DESCRIPTION
**Pull request by Baltoli**

### CODE REVIEW

**Comment by rogersnm** [comment]
LGTM, will wait for travis to finish

File: Lib/src/main/java/uk/ac/cam/cl/wildpetscience/triton/lib/models/AccessData.java [hunk 1]
```diff
@@ -5,13 +5,20 @@
  */
 public class AccessData {
 
-    public final String accessToken; // e.g. "FuRrY"
+    public final String accessToken = "JHsa841fZS91Acv"; // e.g. "FuRrY"
```
Comment (sashforth) [inline Lib/src/main/java/uk/ac/cam/cl/wildpetscience/triton/lib/models/AccessData.java:8]: @rogersnm

Comment (rogersnm) [inline Lib/src/main/java/uk/ac/cam/cl/wildpetscience/triton/lib/models/AccessData.java:8]: public final String accessToken = new BigInteger(130,  new SecureRandom()).toString(32);

(along with associated imports)

**Comment by Baltoli** [comment]
Are we all ok with this?

**Comment by w-shackleton** [comment]
LGTM

**Pull request merged**